index.html

```diff
@@ -361,5 +361,36 @@ <h4>Skills</h4>
 
     </section>
 
+    <section class="contact">
+      <div class="contact-text">
+        <p>
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </p>
+      </div>
+
+      <form action="" class="contact-form" method="post">
```

Comment:
- [ ] [REQUIRED] Good job so far! `Form` should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html), `action` value should be similar to `https://formspree.io/f/{form_id}` but you forgot it implement to your **form's** `action`. Please implement [formspree service](https://formspree.io/html) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/205942111-10b79c5d-e77f-4862-b7c5-1d145207e95d.png)